configure.ac

```diff
@@ -2268,9 +2268,12 @@ AS_CASE([$enable_debug_runtime],
 
 ## Determine how to link with the POSIX threads library
 
+link_gcc_eh=''
```

Comment:
Could this block be moved inside the mingw32 case below? Other platforms won't need to check for symbols in `gcc_eh`, after all.

Reply:
Something tells me that configure macros aren't really meant to be nested by maybe it is workign? Code updated.

Reply:
`AC_CASE` is really a wrapper around shell `case ... in) ... esac`, so no problem here.